.github/CODEOWNERS

```
@@ -666,6 +666,7 @@ peps/pep-0784.rst  @gpshead
 peps/pep-0785.rst  @gpshead
 # ...
 peps/pep-0787.rst  @ncoghlan
+peps/pep-0788.rst  @ZeroIntensity @vstinner
```

I seem to recall we generally just list the sponsor?
```suggestion
peps/pep-0788.rst  @vstinner
```

---

We've listed triagers before. 

The restriction is that we can only list GitHub python org members, so that often excludes authors and can only list core dev sponsors.

But it doesn't exclude triagers. And it's good to list authors where possible: they'll be pinged for review for future changes, which is usually a good thing.